js/apps/system/_admin/aardvark/APP/react/src/views/views/ViewsContext.tsx

```diff
@@ -0,0 +1,33 @@
+import { ViewDescription } from "./View.types";
+import React, { createContext, ReactNode, useContext } from "react";
+import { useFetchViews } from "./useFetchViews";
+
+type ViewsContextType = {
+  views: ViewDescription[] | undefined;
+  isFormDisabled: boolean;
+};
+const ViewsContext = createContext<ViewsContextType>({} as ViewsContextType);
+
+export const ViewsProvider = ({
+  children,
+  isFormDisabled
+}: {
+  children: ReactNode;
+  isFormDisabled?: boolean;
+}) => {
+  const { views } = useFetchViews();
+  return (
+    <ViewsContext.Provider
+      value={{
+        views,
+        isFormDisabled: !!isFormDisabled
+      }}
+    >
+      {children}
+    </ViewsContext.Provider>
+  );
+};
+
+export const useViewsContext = () => {
```

Comment:
I don't see this being used anywhere. I think this whole `ViewsContext` can be removed, as it is unused?